lib/rspec/mocks/message_expectation.rb

```diff
@@ -32,6 +32,8 @@ def verify_messages_received
           end
         end
       end
+
+      def unadvise(_) ; end
```

Comment:
I think I'd prefer this as a two line empty method, do we use this pattern anywhere else?